components/reddit/sources/new-link-on-a-subreddit/new-link-on-a-subreddit.js

```diff
@@ -0,0 +1,76 @@
+const reddit = require("../../reddit.app.js");
+
+module.exports = {
+  key: "new-link-on-a-subreddit",
+  name: "New Link on a subreddit",
+  description: "Emits an event each time a new link is added to a subreddit",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    reddit,
+    subreddit: {
+      type: "string",
+      label: "Subreddit",
+      description: "The subreddit you'd like to watch for new links.",
```

Comment:
I'd recommend changing to:

The subreddit you'd like to watch for new links (if your subreddit is `/r/redditdev`, enter **redditdev** below)

Reply:
ok. changed. yeah defaulting to redditdev was arbitrary -- I thought a dev-related subreddit would be appropriate for Pipedream while being Reddit endorsed forum.